com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1182,6 +1180,32 @@ def __init__(self, rest_client, credentials):
     def _get_url(self, req_name):
         return self._credentials['rest_url'] + self._credentials[req_name]
 
+    def submit_job(self, sab_file, configuration=None):
```

Comment:
Good to have an example, bonus points for showing how a client gets a `Job` object from the return.

Reply:
All of the unit tests show how to get a Job object from the dictionary returned by submit_job(). I could copy some of that code into the method documentation, or just link to the unit test source. Is there a way to generate HTML links in Python documentation?